### PR TITLE
Ryan

### PR DESCRIPTION
**Pull request by xxArcher**

### CODE REVIEW

File: app/controller.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+/* Read package json and return a list of dependencies */
+exports.readPkgJson = function (repoPath) {
+  var pathComponents = repoPath.split("\/");
+  // for (var i=0;i<pathComponents.length;i++){
+  //     console.log(pathComponents[i]);
+  // }
+  return octokit.repos.getContents({
+    owner: pathComponents[3],
+    repo: pathComponents[4],
+    //TODO
```
Comment (meghasinghania22) [inline app/controller.js]: is this still a TODO?

File: app/controller.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  //var repoPath = "https://github.com/MalcolmChen97/React-Native-SmallApps";
+  var pathComponents = repoPath.split("\/");
+  // for (var i=0;i<pathComponents.length;i++){
+  //     console.log(pathComponents[i]);
+  // }
+  console.log("pkgpath: " + pkgJsonPath);
+  return octokit.repos.getContents({
+    owner: pathComponents[3],
+    repo: pathComponents[4],
+    //TODO
```
Comment (meghasinghania22) [inline app/controller.js]: What about this one?

File: app/controller.js (hunk 1 above)
Comment (xxArcher) [reply app/controller.js]: forget this one, just added it.

File: app/controller.js (hunk 2 above)
Comment (xxArcher) [reply app/controller.js]: just added it

**Pull request merged**